src/Movement/Kinematics/HangprinterKinematics.h

```diff
@@ -57,7 +57,8 @@ class HangprinterKinematics : public RoundBedKinematics
 
 private:
 	// Basic facts about movement system
-	static constexpr size_t HANGPRINTER_AXES = 4;
+	const char* ANCHOR_CHARS = "ABCDE";
+	static constexpr size_t HANGPRINTER_MAX_ANCHORS = 5;
 	static constexpr size_t A_AXIS = 0;
```

Comment:
We should add K_AXIS etc here, alternatively remove A_AXIS etc if they are not used anywhere

Reply:
I think we should delete all A_AXIS to D_AXIS constants.
That's the only way to make sure you're lopping to maxAnchors everywhere. But we're not ready yet in that sense like we were in 3.4. There are new uses of A_AXIS since https://github.com/Duet3D/RepRapFirmware/pull/562/files
It's alright, we can get there again. The thing is, newer changes unrelated to this will add whatever is around them. If we loop around ANCHOR_CHARS they will use that adapting it to whatever they're changing. If we keep using these constants, more new uses of them will keep appearing.
I was never ready to remove them in 3.4 though, they were still used in the fastforwards thing.
So until we loop in there we still need them.

Reply:
There are only your pull request going into HangprinterKinematics.h and HangprinterKinematics.cpp for the forseeable future, so we should be able to get rid of A_AXIS to D_AXIS without new uses popping up from elsewhere in the meantime. Let's do it